source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Custom Certificates on the Global CDN
+description: For contract customers who need to retain custom TLS certificates.
+contributors: [ari, alexfornuto]
+earlyaccess: true
+earlynote: A white glove Custom Certificate Concierge Service in now available and limited to  Legacy SSL Customers with a Pantheon contract, including Enterprise, EDU+, Pantheon One, Elite, and Reseller customers.
```

Comment:
Early access note is intended to signal non-standard feature availability, it should not serve as a description for the feature and it shouldn't house eligibility requirements. 

Use the standard earlynote copy to maintain consistent user experience on earlyaccess docs:
> The documentation on this page discusses features and options that are not available across the entire platform.